kf_lib_data_ingest/etl/configuration/base_config.py

```diff
@@ -29,6 +29,10 @@ def __init__(self, filepath, **kwargs):
                 f'{self.config_filepath} failed validation'
             ) from e
 
+    def __getattr__(self, attr):
```

Comment:
This method should prob only in PyModuleConfig bc it will fail for YamlConfig. YamlConfig's contents is a dict

Reply:
Or, you could check the type of contents and if its an attr, then call getattr, if its a dict, then call get?